user-interface/src/data-verification/ConsolidationOrderAccordion.test.tsx

```diff
@@ -353,11 +363,18 @@ describe('ConsolidationOrderAccordion tests', () => {
       status: 'approved',
     };
 
-    vi.spyOn(Chapter15MockApi, 'get').mockResolvedValue({
-      message: '',
-      count: 1,
-      body: [MockData.getAttorneyAssignment()],
-    });
+    // Assigned attorneys and asssociated cases.
+    vi.spyOn(Chapter15MockApi, 'get')
+      .mockResolvedValue({
+        message: '',
+        count: 1,
+        body: [MockData.getAttorneyAssignment()],
+      })
+      .mockResolvedValue({
+        message: '',
+        count: 1,
+        body: [],
+      });
```

Comment:
This test seems to be failing because the attorneys never seem to get loaded.  The approval button will not become enabled until after the attorneys finish loading because it's loading the associated cases.  But since the loading never resolves, the button is never enabled.

Reply:
Yeah, we added a new call to the associated cases endpoint and we have to get the sequence right in the mocks.

Reply:
James replaced this with a more elegant solution.